mne_bids/write.py

```diff
@@ -879,6 +885,24 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
     acquisition, task, run = params['acq'], params['task'], params['run']
     kind = _handle_kind(raw)
 
+    if anonymize:
+        daysback = keep_his = None
+        if isinstance(anonymize, dict):
+            if 'daysback' in anonymize:
+                daysback = anonymize['daysback']
+            if 'keep_his' in anonymize:
+                keep_his = anonymize['keep_his']
+        if ext != '.fif':
+            warn('Converting to FIF to anonymize')
+            ext = '.fif'
+        raw.info['meas_date'] = ((0, 0) if raw.info['meas_date'] is None else
+                                 raw.info['meas_date'])
+        print(raw.info['meas_date'])
+        raw.info = anonymize_info(raw.info, daysback=daysback,
+                                  keep_his=keep_his)
```

Comment:
one more comment -- BIDS recommends that you should [shift to a date before 1900](https://bids-specification.readthedocs.io/en/stable/03-modality-agnostic-files.html#scans-file). The default of MNE is 2000 -- so you need to adjust for that.